src/lib/stores/orderStore.ts

```diff
@@ -0,0 +1,102 @@
+import { PUBLIC_PB_HOST } from "$env/static/public";
+import { writable } from "svelte/store";
+import pocketbase from "pocketbase";
+import { Order } from "$lib/types";
+import type { State } from "$lib/types";
+
+// ref: https://github.com/pocketbase/js-sdk?tab=readme-ov-file#nodejs-via-npm
+import eventsource from "eventsource";
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+(global as any).EventSource = eventsource;
+
+const pb = new pocketbase(PUBLIC_PB_HOST);
+
+const mapToOrder = (data: unknown): Order => {
```

Comment:
Kan man ikke bare bruke data direkte? Det er vel ikke noe poeng i å mappe over til en ny type med samme values. `Order`-klassen var kun midlertidig for å teste. Dette virker spesielt unødvendig da keys har samme navn.

Reply:
I utgangspunktet ville jeg ikke gjort det. Men, nå som vi har `Order`, så fjerner jeg ikke den.

Praksisen er vanlig og produktiv. Likeså, tenker jeg vi kan fikse mer på dette senere. At det blir duplikasjon av navnene er bare slik JS er, beklageligvis. En veldig nyttig ting her er https://zod.dev/.

Dette vil være [parsing](https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/) av `unknown`, selv om vi ikke har definert schema enda.